src/addmult.c

```diff
@@ -364,29 +361,34 @@ int init_and_load_multipliers(void) {
 	return 0;       // couldn't open file
     }
 
-    while (fgets(s_inputbuffer, sizeof(s_inputbuffer), cfp) != NULL) {
+    while ((read = getline(&s_inputbuffer, &s_inputbuffer_len, cfp)) != -1) {
+	if (s_inputbuffer_len > 0) {
+	    if (errno == ENOMEM) {
+		fprintf(stderr, "Error in: %s:%d", __FILE__, __LINE__);
+		perror("RuntimeError: ");
+		exit(EXIT_FAILURE);
+	    }
+	    /* strip leading and trailing whitespace */
+	    g_strstrip(s_inputbuffer);
 
-	/* strip leading and trailing whitespace */
-	g_strstrip(s_inputbuffer);
+	    /* drop comments starting with '#' and empty lines */
+	    if (*s_inputbuffer == '#' || *s_inputbuffer == '\0') {
+		continue;
+	    }
 
-	/* drop comments starting with '#' and empty lines */
-	if (*s_inputbuffer == '#' || *s_inputbuffer == '\0') {
-	    continue;
+	    add_mult_line(s_inputbuffer);
 	}
-
-	add_mult_line(s_inputbuffer);
-
     }
 
     fclose(cfp);
-
+    if (s_inputbuffer > 0)
```

Comment:
Comparing a pointer and 0. It works, but using the typical check
```
if (s_inputbuffer != NULL) 
```
would be clearer.